### PR TITLE
Add `/sig foo` and `/remove-sig foo` commands

### PR DESCRIPTION
**Pull request by spiffxp**
Addresses kubernetes/test-infra#2595

### CODE REVIEW

**Comment by spiffxp** [comment]
/assign @fejta @grodrigues3 
PTAL

**Comment by grodrigues3** [comment]
Seems like this should work.

So I understand, comments are **created** and issues/prs are **opened**?

**Comment by grodrigues3** [comment]
I'll let @fejta give the final LGTM

File: prow/plugins/label/label.go [hunk 1]
```diff
@@ -42,8 +42,8 @@ type assignEvent struct {
 }
 
 var (
-	labelRegex              = regexp.MustCompile(`(?m)^/(area|priority|kind)\s*(.*)$`)
-	removeLabelRegex        = regexp.MustCompile(`(?m)^/remove-(area|priority|kind)\s*(.*)$`)
+	labelRegex              = regexp.MustCompile(`(?m)^/(area|priority|kind|sig)\s*(.*)$`)
```
Comment (fejta) [inline prow/plugins/label/label.go]: Technically seems like we can put this into one labelRegex with `^/(remove-)?(area|priority)` and check whether group(1) is "" or not

I realize this behavior pre-exists though so feel free to ignore (even if you agree)

File: prow/plugins/label/label.go [hunk 2]
```diff
@@ -134,8 +134,8 @@ func (ae assignEvent) getRepeats(sigMatches [][]string, existingLabels map[strin
 }
 
 func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
-	// only parse newly created comments and if non bot author
-	if ae.login == gc.BotName() || ae.action != "created" {
+	// only parse newly created comments/issues/PRs and if non bot author
+	if ae.login == gc.BotName() || !(ae.action == "created" || ae.action == "opened") {
```
Comment (fejta) [inline prow/plugins/label/label.go]: YAY!

**Comment by fejta** [comment]
/lgtm

Click the merge button when you're ready! (Or leave a comment for me to do this)

**Comment by spiffxp** [comment]
@fejta I don't have merge rights, so have at it whenever you're ready.  I'll save the cleanup for a later commit, trying to crawl/walk/run as I grok the codebase

**Comment by grodrigues3** [comment]
@spxtr is this turned on?

**Comment by fejta** [comment]
Yes

**Pull request merged**